stories/Message.stories.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { Message } from '../src/components';
+import { getLipsumSentences } from '../src/mock-data/lipsum';
+
+export default {
+  title: 'Layout|Message',
+  parameters: {
+    purposeFunction: {
+      function: '',
```

Comment:
Are we leaving this empty until it's more developed?

Reply:
I forgot! Now added.